modules/apigw/outputs.tf

```diff
@@ -6,6 +6,14 @@ output "url" {
   value = aws_api_gateway_deployment.deployment.invoke_url
 }
 
+output "url_hostname" {
+  value = element(regex("^https://(.*com)/.*", aws_api_gateway_deployment.deployment.invoke_url), 0)
+}
+
 output "rest_api_id" {
   value = aws_api_gateway_rest_api.api.id
 }
+
+output "stage" {
```

Comment:
An APIGW has multiple stages so not sure it makes sense to have a single `stage` output variable. The `stage` is actually set as an input variable to the module so I don't think we need to output it.

Reply:
No those are good points though, it's confusing there's one stage variable when this could change (or we could define `stages`) 

Actually, there's no need to add this because initially, I didn't know how to reuse input variables in the [impersonation-gateway repo](https://github.com/efcloud/el-b2c-school-impersonation-gateway) (which was the reason for this), but I've now learnt the existence of the 'Local' type, so I don't think I have a reason of including this anymore